### DISCUSSION TITLE
build creates many files in my git, what can I do?

### DISCUSSION THREAD

**Discussion by octaeder** [texstudio-org/texstudio#2409] [Q&A] [answered]
Hi,
I use msys2 as described in the wiki and github desktop to build dev versions of TeXstudio. After running make -j 8 github desktop locates over 400 new files:

![grafik](https://user-images.githubusercontent.com/102688820/173373093-fa699415-db2e-4764-8568-048085cccd69.png)

Less than 5 of them have been changed by me. What is the best way to proceed? Github desktop comes with a small .gitignore file, obviously not with suitable entries (like .moc, since files are named moc_*). But where do I know from, which patterns to use, so I don't miss anything? Is there a way to split between repo sources and objects creates by the build? I think last idea would be better.

Thanks!

**Comment by sunderme** [comment, marked as answer]
simplest solution is to build in a different directory.
qtcreator does so by default. Just call qmake .../texstudio.pro from a different directory.

**Reply by octaeder** [reply]
Yes, that's the hint I needed (and I think you wanted to say qmake6). thx

**Reply by octaeder** [reply]
do I have to rerun qmake6 after every code change (for source update) or is make -j 8 enough as I'm used to do so?

**Reply by sunderme** [reply]
qmake needs to be run only once. Usually make handles all changes.
